framework/src/main/java/org/tron/core/services/http/Util.java

```diff
@@ -499,6 +500,12 @@ public static byte[] getAddress(HttpServletRequest request) throws Exception {
     byte[] address = null;
     String addressParam = "address";
     String addressStr = request.getParameter(addressParam);
+
+    if (APPLICATION_FORM_URLENCODED.equals(request.getContentType())
```

Comment:
It's not clear. Is this the only scene?

Reply:
It is recommended to make a clear distinction between GET, POST (application/json, application/x-www-form-urlencoded)

Reply:
There is no need to add other types of content types, and these types are supported even if other types of judgments are not explicitly added.